Bottom: 27d628b613db49b958d4ec1735eab8ef193802a9
Top:    4def70162dc07e26b63482e176ee22b7f48020fb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-14 21:40:40 +0100

Refresh of gtk-fix-ghid_zoom_view_fit-to-

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 2fa4aa3..488f971 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -128,9 +128,10 @@ ghid_zoom_view_rel (Coord center_x, Coord center_y, double factor)
 static void
 ghid_zoom_view_fit (void)
 {
-  ghid_pan_view_abs (0, 0, 0, 0);
-  ghid_zoom_view_abs (0, 0, MAX (PCB->MaxWidth  / gport->width,
-                                 PCB->MaxHeight / gport->height));
+  ghid_pan_view_abs (SIDE_X (0), SIDE_Y (0), 0, 0);
+  ghid_zoom_view_abs (SIDE_X (0), SIDE_Y (0),
+                      MAX (PCB->MaxWidth  / gport->width,
+                           PCB->MaxHeight / gport->height));
 }
 
 static void
